scheduler/src/main/java/gov/nasa/jpl/aerie/scheduler/constraints/TimeRangeExpression.java

```diff
@@ -123,7 +123,7 @@ public void setName(String name) {
   protected String name = "TRE_" + Math.abs(new Random().nextInt());
   protected List<TimeRangeExpression> timeRangeExpressions;
   protected List<Object> filtersAndTransformers;
-  protected List<StateConstraintExpression> stateExpr;
+  public List<StateConstraintExpression> stateExpr;
```

Comment:
Why not an accessor?

Reply:
Updated to use an accessor :+1:

https://github.com/NASA-AMMOS/aerie/blob/4fffd68931d5a1160ae6abcefeae068de2258711/scheduler/src/main/java/gov/nasa/jpl/aerie/scheduler/constraints/TimeRangeExpression.java#L130-L132